packages/app/components/Queue/Student/StudentQuestionRephraseModal.tsx

```diff
@@ -0,0 +1,36 @@
+import { RephraseQuestionPayload } from "@koh/common";
+import { Button } from "antd";
+import Modal from "antd/lib/modal/Modal";
+import React, { ReactElement } from "react";
+
+type StudentQuestionRephraseModalProps = {
+  courseId: number;
+  payload: RephraseQuestionPayload;
+  handleClose: (number) => void;
+};
+export default function StudentQuestionRephraseModal({
+  courseId,
+  payload,
+  handleClose,
+}: StudentQuestionRephraseModalProps): ReactElement {
+  return (
+    <Modal
+      visible={true}
+      closable={false}
+      footer={[
+        <Button
```

Comment:
```suggestion
        <Link 
            href={{ pathname: "/course/[cid]/queue/[qid]" query: { edit_question: 'true' } }} 
            as={`/course/${courseId}/queue/${payload.queueId}`}
            >
            <Button
```

Reply:
When I tried this, I didn't trigger the edit question modal